curp/src/rpc/connect.rs

```diff
@@ -468,6 +467,119 @@ impl InnerConnectApi for Connect<InnerProtocolClient<Channel>> {
     }
 }
 
+/// A connect api implementation which bypass kernel to dispatch method directly.
```

Comment:
It uses direct function calls instead of going through the TCP/IP protocol stack (check the loopback address in the linux kernel), which called `bypass kernel`